core/arch/arm/plat-imx/a7_plat_init.S

```diff
@@ -65,3 +66,12 @@ UNWIND(	.fnstart)
 	bx	lr
 UNWIND(	.fnend)
 END_FUNC plat_cpu_reset_early
+
+FUNC get_core_pos , :
```

Comment:
It's `get_core_pos_mpidr()` that should be overridden

Reply:
Seems, I missed the changes you made about __get_core_pos. Thanks. I'll fix in v2.